src/etc/inc/openvpn.inc

```diff
@@ -1096,8 +1106,32 @@ function openvpn_reconfigure($mode, $settings) {
 					$tls_directive = "tls-auth";
 					if ($mode == "server") {
 						$tlsopt = 0;
+						//switch($settings['tlsauth_keydir']){
```

Comment:
yes they are there for the server side, but the server page is ongoing. Also there should be a way to correlate client/server choosen parameter to always use the right one.
Of course this is not possible for those clients you do not have control the corresponding server' otherwise one can do the correlation.